mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -2,6 +2,22 @@
 
 # Deploy the management hub services (agbot, exchange, css, sdo, postgre, mongo), the agent, and the CLI on the current host.
 
+# Get current hardware architecture
+export ARCH=$(uname -m | sed -e 's/aarch64.*/arm64/' -e 's/x86_64.*/amd64/' -e 's/armv.*/arm/')
+if [ "${ARCH}" = "ppc64le" ]; then
+    export ARCH_DEB=ppc64el
+else
+    export ARCH_DEB="${ARCH}"
+fi
+
+# Set the correct default value for docker-compose command
+if [ -f /usr/bin/docker-compose ]; then
```

Comment:
This check is correct because it defines default value for variable `DOCKER_COMPOSE_CMD` that masks difference in the two `Docker Compose` versions: Binary or Python-based. It is done for Management functions of `deploy-mgmt-hub.sh` script (see its flags).
If `Docker Compose` is not installed yet and check return `false` it is not a problem because it doesn't affect code below. When Docker Compose is installed firstly this variable `DOCKER_COMPOSE_CMD` is redefined correctly in dependence of version used.

Reply:
Closed because outdated and duplicated

Reply:
Closed as outdated and duplicated.